src/runtime_src/core/tools/xbmgmt2/xbmgmt

```diff
@@ -10,6 +10,16 @@ fi
 # Working variables
 XRT_PROG=xbmgmt
 
+# Determine which version of xbmgmt should be used
+# Check to see environment variable is NOT set
+if [ -z ${XRT_TOOLS_NEXTGEN+x} ]; then 
+  XRT_PROG=xbmgmt
+else
+  if [ "${XRT_TOOLS_NEXTGEN,,}" = "true" ]; then
```

Comment:
isn't this called 'elif' ?

Reply:
Yes, it is....I will update it.

Regarding the new ENV var and --new, if the user specified the "--new" option, it will (by design) take precedence over the variable name.

Reply:
Done